

```python

```

### I. Effect of Genetic Variants on ASD Risk

1. Protein Truncating Variants (PTV), missense variants and synonymous variants shows greater burden in ASD <br>
2. To understand functional severity with higher burden **‘probability of loss-of-function intolerance’’ (pLI) score** and **‘‘missense badness, PolyPhen-2, constraint’’ (MPC) score** was introduced.

Divided the list of rare autosomal genetic variants into seven tiers of predicted functional severity: *Fig 1A*:
- Three tiers for PTVs by pLI score (=> 0.995, 0.5–0.995, 0–0.5) in order of decreasing expected effect
- Three tiers for missense variants by MPC score (=>2, 1–2, 0–1) 
- Single tier for synonymous variants, expected to have minimal effect

We further divided variants by their inheritance pattern: 
- de novo
- inherited 
- case-control

- ASD is associated with reduced fecundity, variation associated with it is subject to natural selection.<br>
- Inherited variation has survived at least one generation of viability and fecundity selection in the parental generation whereas de novo variation in offspring has not.<br>

- Thus, on average, de novo mutations are exposed to less selective pressure and could mediate substantial risk for ASD. <br>
- This expectation is borne out by the substantially higher proportions of all three PTV tiers and the two most severe missense variant tiers in de novo compared with inherited variants

<img src='img2/bauxbam_img1.jpg'>

**Figure 1. Distribution of Rare Autosomal Protein-Coding Variants in ASD Cases and Controls** <br>
(A) The proportion of rare autosomal genetic variants split by predicted functional consequences, represented by color, is displayed for family-based (split into denovo and inherited variants) and case-control data. PTVs and missense variants are split into three tiers of predicted functional severity, represented by shade, based on the pLI and MPC metrics, respectively.<br>

(B) The relative difference in variant frequency (i.e., burden) between ASD cases and controls (top and bottom) or transmitted and untransmitted parental variants (center) is shown for the top two tiers of functional severity for PTVs (left and center) and the top tier of functional severity for missense variants (right). Next to the bar plot, the same data are shown divided by sex.<br>

(C) The relative difference in variant frequency shown in (B) is converted to a trait liability Z score, split by the same subsets used in (A). For context, a Z score of 2.18 would shift an individual from the population mean to the top 1.69% of the population (equivalent to an ASD threshold based on 1 in 68 children; Christensen et al., 2016). No significant difference in liability was observed between males and females for any analysis.
Statistical tests: (B) and (C), binomial exact test (BET) for most contrasts; exceptions were ‘‘both’’ and ‘‘case-control,’’ for which Fisher’s method for combining BET p values for each sex and, for case-control, each population was used; p values corrected for 168 tests are shown

### Sex Differences in ASD Risk
1. ASD is more prevalent in males than females.

2. Risk variation has larger effects in males than in females so that females require a higher burden to reach the same diagnostic threshold as males.
3.  we observe a 2-fold enrichment of de novo PTVs in highly constrained genes in affected females (n = 1,097) versus affected males (n = 5,333)

### ASD Gene Discovery

1. From family data, we include de novo PTVs as well as de novo missense variants, whereas from the case-control, we include only PTVs; we do not include inherited variants because of the limited liabilities observed

2. Of the 102 ASDassociated genes, 60 were not discovered by our earlier analyses

3. These include 30 considered truly novel because they have not been implicated in autosomal dominant neurodevelopmental disorders (ASD, developmental delay, epilepsy, and intellectual disability) and were not significantly enriched for denovo and/or rare variants in previous studies



<img src=img2/bauxbam_img2.jpg>

**Figure 2. Gene Discovery in the ASC Cohort** <br>
(A) WES data from 35,584 samples are entered into a Bayesian analysis framework (TADA) that incorporates pLI score for PTVs and MPC score for missense variants.<br>

(B) The model identifies 102 autosomal genes associated with ASD at a false discovery rate (FDR) threshold of 0.1 or less, which is shown on the y axis of this Manhattan plot, with each point representing a gene. Of these, 78 pass the threshold FDR of 0.05 or less, and 26 pass the threshold family-wise error rate (FWER) of 0.05 or less.<br>

(C) Repeating our ASD trait liability analysis (Figure 1C) for variants observed within the 102 ASD-associated genes only.<br>

### Genetic Characterization of ASD Genes

<img src=img2/bauxbam_img3.jpg>

**Figure 3. Genetic Characterization of ASD Genes** <br>
(A) Count of PTVs versus missense variants (MPC >= 1) in cases for each ASD-associated gene (red points, selected genes labeled). These counts reflect the data used by TADA for association analysis: de novo and case-control data for PTVs; de novo only for missense.

(B) Location of ASD de novo missense variants in DEAF1. The five ASD variants (marked in red) are in the SAND (Sp100, AIRE-1, NucP41/75, DEAF-1) DNAbinding domain (amino acids 193–273, spirals show a helices, arrows show b sheets, KDWK is the DNA-binding motif) alongside 10 variants observed in NDD, several of which have been shown to reduce DNA binding, including Q264P and Q264R (Chen et al., 2017; Heyne et al., 2018; Vulto-van Silfhout et al., 2014).

(C) Location of ASD missense variants in KCNQ3. All four ASD variants are located in the voltage sensor (fourth of six transmembrane domains), with three in the same residue (R230), including the gain-of-function R230C mutation observed in NDD (Heyne et al., 2018; Miceli et al, 2015). Five inherited variants observed in benign infantile seizures are shown in the pore loop (Landrum et al., 2014; Maljevic et al., 2016).

(D) Location of ASD missense variants in SCN1A alongside 17 de novo variants in NDD and epilepsy (Heyne et al., 2018).

(E) Location of ASD missense variants in SLC6A1 alongside 31 de novo variants in NDD and epilepsy (Heyne et al., 2018; Johannesen et al., 2018).

(F) Subtelomeric 2q37 deletions are associated with facial dysmorphisms, brachydactyly, high BMI, NDD, and ASD (Leroy et al., 2013). Although three genes within the locus have a pLI score of 0.995 or higher, only HDLBP is associated with ASD.

(G) Deletions at the 11q13.2q13.4 locus have been observed in NDD, ASD, and otodental dysplasia (Coe et al., 2014; Cooper et al., 2011). Five genes within the locus have a pLI score of 0.995 or higher, including two ASD genes: KMT5B and SHANK2.

(H) Assessment of gene-based enrichment, via MAGMA, of 102 ASD genes against genome-wide significant common variants from six GWASs.

(I) Gene-based enrichment of 102 ASD genes in multiple GWASs as a function of effective cohort size. The GWAS used for each disorder in (I) has a black outline.

Statistical tests: (F) and (G), TADA; (H) and (I), MAGMA

### Patterns of Mutations in ASD Genes

The ratio of PTVs to missense mutations varies substantially between genes (Figure 3A). Some genes reach our association threshold through PTVs alone (e.g., ADNP), and three genes have a significant excess of PTVs relative to missense mutations, accounting for gene mutability: SYNGAP1, DYRK1A, and ARID1B. Because of the increase cohort size and availability of the MPC metric, we are also able, for the first time, to associate genes with ASD based primarily on de novo missense variation.

Four genes carry four or more de novo missense variants (MPC >= 1) in ASD cases and one or no PTVs: DEAF1, KCNQ3, SCN1A, and SLC6A1

- DEAF1, 5 de novo missense variants :: critical for dimerization and DNA binding
- KCNQ3, all four de novo missense variants modify arginine residues in the voltage-sensing fourth transmembrane domain, with three at a single residue previously characterized as gain of function in NDD (neurodevelopmental delay)
- SCN1A, 4 de novo missense variants, seizures.
- SLC6A1, 8 de novo missense variants , seizures.


### Relationship of ASD Genes with GWAS Signals

Recent genome-wide association studies (GWASs) reveal a handful of ASD-associated loci. Notably, among the five GWAS-significant ASD hits, KMT2E is implicated by both GWAS and the list of 102

- Using MAGMA (multi-marker analysis of genomic annotation; de Leeuw et al., 2015), whether common genetic variation in or near the 102 identified genes (within 10 kb) influences ASD risk or other related traits. 

-For these associated genes, MAGMA integrates GWAS summary statistics to determine whether their signal is enriched over background; namely, brainexpressed protein-coding genes. 

six GWAS datasets: ASD, schizophrenia, major depressive disorder, and attention deficit hyperactivity disorder (ADHD), which are all positively genetically correlated with ASD and with each other; educational attainment, which is positively correlated with ASD and negatively correlated with schizophrenia and ADHD; and human height as a negative control

-only the schizophrenia and educational attainment GWAS signals show significant enrichment in ASD genes (Figure 3H). 

The ASD GWAS signal was not enriched, potentially because common and rare variation contributing to ASD risk affect distinct genes or potentially because we currently lack the sample sizes to detect the convergence of the two. We conjecture that the second hypothesis is more likely because of three results: the known genetic correlation of schizophrenia and educational attainment with ASD, the enrichment of common variation conferring risk for both found in the 102 ASD genes, and the statistically significant overlap we demonstrate for KMT2E. 

In addition, effective cohort sizes for schizophrenia, educational attainment, and height dwarf that for ASD (Figure 3I), and the quality of the GWAS signal strongly increases with sample size. Thus, for results from well-powered GWASs, it is reassuring that there is no signal for height but a clearly detectable signal for two traits genetically correlated with ASD.

### Relationship between ASD and Other Neurodevelopmental Disorders

Family studies yield high heritability estimates in ASD (Yip et al.,2018), whereas estimates of heritability in severe NDD are lower (Reichenberg et al., 2016). Consistent with these observations, 

- Previous exome studies identify a higher frequency of disruptive denovo variants in severe NDD than in ASD (Deciphering Developmental Disorders Study, 2017). 


- Because 30%–50% of ASD individuals have comorbid intellectual disability and/or NDD, many genes are associated with both disorders (Pinto et al., 2010). 

Distinguishing genes that, when disrupted, lead to ASD more frequently than NDD could shed new light on how atypical neurodevelopment maps onto the core deficits of ASD.<br>

To partition the 102 ASD genes in this manner, we compiled data from 5,264 trios ascertained for severe NDD (Table S4) and compared the relative frequency, R, of disruptive de novo variants (which we define as PTVs or missense variants with MPC => 1) in ASD- or NDD-ascertained trios.

- Genes with R > 1 were classified as ASD-predominant (ASDP, 53 genes)

- Genes with R < 1 were classified as ASD with NDD (ASDNDD, 49 genes).



### Functional Dissection of ASD Genes
Past analyses have identified two major functional groups of ASD genes: 
- Those involved in gene expression regulation (GER)
- Those involved in neuronal communication (NC).

Similarly, Gene Ontology enrichment analysis with the 102 ASD genes identifies 

- 16 genes in the ‘‘regulation of transcription from RNA polymerase II promoter’’ category (GO:0006357, 5.7-fold enrichment, FDR = 6.2 x 3 10E-6) 

- 9 genes in the ‘‘synaptic transmission’’ category (GO:0007268, 5.0-fold enrichment, FDR = 3.8 x 3 10E-3). 

Combined Gene Ontology and primary literature to assign genes to GER (n = 58), NC (n = 24), ‘‘cytoskeleton organization’’ (n = 9, GO:0007010), or ‘‘other’’ categories.<br>

Interestingly, ASD subjects who carry disruptive de novo variants in either GER or NC genes showed delayed age of walking and reduced IQ compared with those with no mutations in the 102 genes. <br>

Carriers of disruptive variants in GER genes show significantly greater delays in age of walking compared with those with disruptive variants in NC gene

### ASD Genes Are Expressed Early in Brain Development
The 102 ASD genes can be subdivided by phenotypic effect (53 ASDP genes, 49 ASDNDD genes) and functional role (58 GER genes, 24 NC genes) to give five gene sets. (Including all 102)<br>

First evaluated enrichment of these five gene sets in the 53 tissues with bulk RNA sequencing (RNA-seq) data in the Genotype-Tissue Expression (GTEx) resource (Battle et al., 2017). <br>

Enrichment was observed in 11 of 13 brain regions, with the strongest enrichment in: <br>
- the cortex (30 genes, p = 3x10E-6, Figure 5A)
- cerebellar hemisphere (48 genes, p = 3x10E-6, Figure 5A). 
- Of the four gene subsets, NC genes were the most highly enriched in the cortex (17 of 23; Figure 5A), whereas GER genes were the least enriched (10 of 58, Figure 5A). 

- Applying unsupervised co-expression network analysis (weighted gene co-expression network analysis; WGCNA) to the BrainSpan gene expression data yielded enrichment for cortically-expressed ASD genes within discretely co-expressed groups of genes (i.e., modules) across developmen; however, GER and NC genes co-clustered separately (Figure S4). 

- Thus, in keeping with prior analyses, ASD genes are expressed at high levels in the human cortex and early in development. 

- The differing expression patterns of GER and NC genes could reflect two distinct periods of ASD susceptibility during development or a single susceptibility period when both functional gene sets are highly expressed in mid-to-late fetal development.

### ASD Genes Are Enriched in Maturing Inhibitory and Excitatory Neurons

- Here we perform a more direct assessment, examining expression of the 102 ASD-associated genes in an existing single-cell RNA-seq dataset of 4,261 cells from the prenatal human forebrain.

- They identified 25 clusters of data (t-distributed stochastic neighbor embedding (t-SNE) analysis)

- ASD genes are enriched in maturing and mature neurons of excitatory and inhibitory line ages (Figures 5F and 5G)
 
- Early excitatory neurons (C3) expressed the most ASD genes (72; OR = 5.0, p < 1 3 1010, Fisher’s exact test [FET]), whereas the choroid plexus (C20) and microglia (C19) expressed the fewest (39; p = 0.09 and 0.14, respectively; FET); 
- 14 genes were not expressed in any cluster (Figure 5G).

- Within the major neuronal lineages, early excitatory neurons (C3) and striatal interneurons (C1) showed the greatest degree of enrichment (72 and 51 genes, respectively; p < 1 3 1010, FET; Figures 5F and 5G; Table S5).

- Overall, maturing and mature neurons in the excitatory and inhibitory lineages showed a similar degree of enrichment, whereas the excitatory lineage expressed the most ASD genes, paralleling the larger numbers of genes expressed in excitatory lineage cells (Figure 5H).

- To validate the tSNE clusters, we selected 10% of the expressed genes showing the greatest variability among the cell types and performed hierarchical clustering (Figure 5I). This recaptured the division of these clusters by lineage (excitatory versus inhibitory) and by development stage (radial glia and progenitors versus neurons)

### Prediction of Novel Risk Genes and Functional Relationships among ASD Genes

- Genes that are co-expressed with these ASD genes, interact with them, or are regulated by them could lend insight into convergent or auxiliary functions related to risk. 

- In particular, we examined whether in silico network analyses would highlight additional risk genes and clarify the regulatory relationships between GER and NC genes.

- Three additional analyses were performed: the discovering association with networks (DAWN) approach to integrate TADA scores and gene co-expression data, enrichment analysis using protein-protein interaction (PPI) networks, and analyses using results from chromatin and cross-linked immunoprecipitation sequence assays to evaluate regulatory networks (STAR Methods; Figure S5; Table S5). 

- Using the TADA results and BrainSpan gene co-expression data from the midfetal human cortex, DAWN yields 138 genes (FDR % 0.005), including 83 genes that are not captured by TADA, with 69 of these 83 correlated with many other genes. 

- Notably, 12 of the genes DAWN previously predicted as plausibly contributing to risk (De Rubeis et al., 2014) were identified as new TADA genes here (enrichment p = 8.4 3 1011; OR = 16.4).

- To explore whether GER and NC gene sets interact more than would be expected by chance, we analyzed PPI networks and found that they do not; there was an excess of interactions among all ASD genes (82 genes, p = 0.02, FET), GER genes (49 genes, p = 0.006), and NC genes (12 genes, p = 0.03) but not among GER and NC genes (2 genes,
p = 1.00). 
- GER genes did not regulate the NC genes, according to our analyses, although GER-GER regulation was enriched. Even CHD8, a prominent and well-characterized ASD GER gene, did not regulate NC genes more than expected by chance (Figure S5).

Measures of functional severity assessing evolutionary constraint against deleterious genetic variation, such as the
‘‘probability of loss-of-function intolerance’’ (pLI) score (Kosmicki et al., 2017; Lek et al., 2016) and the integrated ‘‘missense badness, PolyPhen-2, constraint’’ (MPC) score (Samocha et al., 2017), can further delineate variant classes with higher burden

<img src='img2/bauxbam_img5.jpg'>



**Figure 5. Analysis of 102 ASD-Associated Genes in the Context of Gene Expression Data**

(A) GTEx bulk RNA-seq data from 53 tissues were processed to identify genes enriched in specific tissues. Gene set enrichment was performed for the 102 ASD genes and four subsets (ASDP, ASDNDD, GER, and NC) for each tissue. Five representative tissues are shown here, including cortex, which has the greatest degree of enrichment (OR = 3.7; p = 2.6 X 10E-6).

(B) BrainSpan bulk RNA-seq data across 10 developmental stages was used to plot the normalized expression of the 101 cortically expressed ASD genes (excluding PAX5, which is not expressed in the cortex) across development, split by the four subsets.

(C) A t-statistic was calculated, comparing prenatal with postnatal expression in the BrainSpan data. The t-statistic distribution of 101 ASD-associated genes shows a prenatal bias (p = 8 3 108) for GER genes (p = 9 3 1015), whereas NC genes are postnatally biased (p = 0.03).

(D) The cumulative number of ASD-associated genes expressed in RNA-seq data for 4,261 cells collected from human forebrain across prenatal development (Nowakowski et al., 2017).

(E) t-SNE analysis identifies 19 clusters with unambiguous cell type in these single-cell expression data.

(F) The enrichment of the 102 ASD-associated genes within cells of each type is represented by color. The most consistent enrichment is observed in maturing and mature excitatory (bottom center) and inhibitory (top right) neurons.

(G) The developmental relationships of the 19 clusters are indicated by black arrows, with the inhibitory lineage shown on the left (cyan), excitatory lineage in the middle (magenta), and non-neuronal cell types on the right (gray). The proportion of the 102 ASD-associated genes observed in at least 25% of cells within the cluster is shown by the pie chart, whereas the log-transformed Bonferroni-corrected p value of gene set enrichment is shown by the size of the red circle.

(H) The relationship between the number of cells in the cluster (x axis) and the p value for ASD gene enrichment (y axis) is shown for the 19 cell type clusters. Linear regression indicates that clusters with few expressed genes (e.g., C23 newborn inhibitory neurons) have higher p values than clusters with many genes (e.g., C25 radial glia).

(I) The relationship between the 19 cell type clusters using hierarchical clustering based on the 10% of genes with the greatest variability among cell types.


```python

```
